Makefile

```diff
@@ -333,9 +333,13 @@ else
 	cp $(GOPATH)/bin/windows_amd64/platform.exe $(DIST_PATH)/bin # from cross-compiled bin dir
 endif
 	@# Package
-	tar -C dist -czf $(DIST_PATH)-$(BUILD_TYPE_NAME)-windows-amd64.tar.gz mattermost
+ifeq (, $(shell which zip))
```

Comment:
Not sure I like the fallback. I would much rather have it fail then produce inconsistent output. The `make package` target is designed for our build machine and I would much rather have it fail here then when it's trying to upload.